web_timeline/static/src/js/web_timeline.js

```diff
@@ -0,0 +1,564 @@
+                } else {
+                    this.do_switch_view('form', null, { mode: "view" });
+                }
+            }
+            else {
+                var id_cast = parseInt(id).toString() == id ? parseInt(id) : id;
+                var pop = new instance.web.form.FormOpenPopup(self);
+                pop.on('write_completed', self, self.reload);
+                pop.show_element(
+                    self.dataset.model,
+                    id_cast,
+                    null,
+                    {readonly: true, title: title}
+                );
+                var form_controller = pop.view_form;
+                form_controller.on("load_record", self, function() {
+                     var footer = pop.$el.closest(".modal").find(".modal-footer");
+                     footer.find('.oe_form_button_edit,.oe_form_button_save').remove();
+                     footer.find(".oe_form_button_cancel").prev().remove();
+                     footer.find('.oe_form_button_cancel').before("<span> or </span>");
+                     button_edit = _.str.sprintf("<button class='oe_button oe_form_button_edit oe_bold oe_highlight'><span> %s </span></button>",_t("Edit"));
+                     button_save = _.str.sprintf("<button class='oe_button oe_form_button_save oe_bold oe_highlight'><span> %s </span></button>",_t("Save"));
+                     footer.prepend(button_edit + button_save);
+                     footer.find('.oe_form_button_save').hide();
+                     footer.find('.oe_form_button_edit').on('click', function() {
+                         form_controller.to_edit_mode();
+                         footer.find('.oe_form_button_edit,.oe_form_button_save').toggle();
+                     });
+                     footer.find('.oe_form_button_save').on('click', function() {
+                         form_controller.save();
+                         form_controller.to_view_mode();
+                         footer.find('.oe_form_button_edit,.oe_form_button_save').toggle();
+                     });
+                     var chatter = pop.$el.closest(".modal").find(".oe_chatter");
+                     if(chatter.length){
+                         var chatter_toggler = $($.parseHTML(_.str.sprintf('<div class="oe_chatter_toggle fa fa-plus-circle"><span> %s </span><div class="oe_chatter_content"></div></div>', _t("Messages"))));
+                         chatter.before(chatter_toggler)
+                         var chatter_content = chatter_toggler.find(".oe_chatter_content");
+                         chatter_content.prepend(chatter); 
+                         chatter_content.toggle();
+                         chatter_toggler.click(function(){
+                             chatter_content.toggle();
+                             chatter_toggler.toggleClass('fa-plus-circle fa-minus-circle');
+                         });
+                     }
+                });
+            }
+            return false;
```

Comment:
This doesn't refresh the view. You can edit for example the title of the element